### PR TITLE
change $arHeader['Content-type'] to array

### PR DESCRIPTION
**Pull request by ickbinhier**
with $arHeader['Content-type'] = ''; become i this errors:

PHP Warning:  Illegal string offset 'type' in /lib/Cws/MailBounceHandler/Handler.php on line 915
PHP Warning:  Illegal string offset 'report-type' in /lib/Cws/MailBounceHandler/Handler.php on line 918
PHP Warning:  Illegal string offset 'boundary' in /lib/Cws/MailBounceHandler/Handler.php on line 918
PHP Warning:  Illegal string offset 'type' in /lib/Cws/MailBounceHandler/Handler.php on line 1111
PHP Notice:  Undefined index: first in /lib/Cws/MailBounceHandler/Handler.php on line 589

### CODE REVIEW

File: lib/Cws/MailBounceHandler/Handler.php [hunk 1]
```diff
@@ -911,7 +911,7 @@ private static function parseHeader($arHeader)
 
         if (isset($arHeader['Content-type'])) {
             $ar_mr = explode(';', $arHeader['Content-type']);
-            $arHeader['Content-type'] = '';
+            $arHeader['Content-type'] = [];
```
Comment (crazy-max) [inline lib/Cws/MailBounceHandler/Handler.php]: Hi,

Thanks for this PR but can you make a new commit by replacing `[]` with `array()` for backward compatibility

**Comment by crazy-max** [comment]
Hi,

Are you on PHP 7.x ?

**Comment by ickbinhier** [comment]
PHP 7.1

**Comment by ickbinhier** [comment]
i have changed to array()

**Pull request merged**